system_test/common/helpers/aest_api.erl

```diff
@@ -387,6 +390,25 @@ sc_wait_open_(IConn, RConn, Type) ->
 
     ok.
 
+sc_wait_close(Channel) ->
+    #{ initiator := {_IAccount, IConn}
+     , responder := {_RAccount, RConn} } = Channel,
+    case sc_wait_for_channel_event(IConn, info) of
+        {ok, #{ <<"event">> := <<"closing">> }} ->
+            {ok, #{ <<"event">> := <<"closed_confirmed">> }} = sc_wait_for_channel_event(IConn, info),
+            {ok, #{ <<"event">> := <<"died">> }} = sc_wait_for_channel_event(IConn, info);
+        {ok, #{ <<"event">> := <<"close_mutual">> }} ->
```

Comment:
It looks like we have changed the sequence of events for `initiator` for Lima? This should be further investigated